Source/Core/Common/ChunkFile.h

```diff
@@ -235,7 +235,7 @@ class PointerWrap
 
 		while (true)
 		{
-			u8 shouldExist = (list_cur ? 1 : 0);
+			u8 shouldExist = !!list_cur;
```

Comment:
When in doubt, typeid: http://ideone.com/0MzWhW

Run them through `c++filt -t <output here>` for fully formed names.